### DISCUSSION TITLE
Visual Studio 2019

### DISCUSSION THREAD

**Discussion by evalen43** [krvajal/vscode-fortran-support#903] [Q&A - deprecated see Fortran Discourse]
I installed the fpm extension in Visual Studio 2019 to work with FORTRAN. It does not work! When entering the compiler parameters and clicking okay, Visual Studio crushes and reloads, losing the entries.

**Comment by gnikit** [comment]
Hi @evalen43 this is the repository for Modern Fortran for VS Code extension, see [here](https://marketplace.visualstudio.com/items?itemName=fortran-lang.linter-gfortran), we don't author or maintain any code for Visual Studio or for an fpm extension.

We are going to add build and test support in VS Code and fpm, but this project is probably not in the books for this year.

Given that the extension was made by the Sourcery Institute, maybe @everythingfunctional can help.

**Comment by everythingfunctional** [comment]
Hi @evalen43 , I made that as a prototype for a client that didn't end up using it. I haven't even tried to install and load it in probably over a year. I don't know how easy it would be to fix, and I haven't really seen much interest from people in using it. I don't really have much time to spend on it either. Sorry it didn't work for you, but it's open source and permissively licensed so you're welcome to try and fix it if you want.
